### DISCUSSION TITLE
Feature request: Make POIs (e.g. supermarkets) findable in combination with the town

### DISCUSSION THREAD

**Discussion by clamyahoo** [osmandapp/OsmAnd#20703] [Ideas]
Hi,
I would like to suggest an improvement in the search function: Especially with shops, I would like to be able to just write the name and the town where they are into the search field. For example, there is a REWE supermarket in Vogesenallee 33, 77694 Kehl (it could be anything, Aldi, Edeka, Lidl and other sorts of great supermarkets). In Nominatim I just have to write REWE Kehl and it shows me their three supermarkets in Kehl. In OsmAnd, I can only search by the name or the brand or maybe by the category shop > supermarket. But then it only shows the distances to the next ones or I have to search on the map. This is quite bad. And at the same time, if I look for street, it works, for example if I search "Hauptstrasse Kehl" it shows me the Hauptstraße in Kehl. Would that be possible, e.g. to combine just the name or the brand of an object and the town?
Kind regards,
Clemens

**Comment by dmpr0** [comment]
Currently in progress https://github.com/osmandapp/OsmAnd/issues/2993
